misc/hamming_distance.c

```diff
@@ -0,0 +1,62 @@
+/**
+ * @file
+ * @brief [Hamming distance](https://en.wikipedia.org/wiki/Hamming_distance)
+ * algorithm implementation.
+ * @details
+ * In information theory, the Hamming distance between two strings of
+ * equal length is the number of positions at which the corresponding symbols
+ * are different.
+ * @author [Aybars Nazlica](https://github.com/aybarsnazlica)
+ */
+
+#include <assert.h>
+#include <stdio.h>
```

Comment:
Please add a one-line description of what the library/header is for (**see the example below**)
```c
#include <stdio.h>     /// for IO operations
#include <assert.h>    /// for assert
```

Reply:
Fixed, thanks.